tools/packaging/packaging.psm1

```diff
@@ -408,73 +402,28 @@ function New-UnixPackage {
             "libc6",
             "libcurl3",
             "libgcc1",
-            "libssl1.0.0",
+            "libgssapi-krb5-2",
```

Comment:
The package changes have been tested with currently supported platforms at least?

Reply:
I have tested the DEB package on Ubuntu.16.04 and Ubuntu.14.04, but not on Debian 8 and Debian 9 yet.